leggend/api/models/accounts.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+from typing import List, Optional, Dict, Any
+
+from pydantic import BaseModel
+
+
+class AccountBalance(BaseModel):
+    """Account balance model"""
+    amount: float
+    currency: str
+    balance_type: str
+    last_change_date: Optional[datetime] = None
+    
+    class Config:
```

Comment:
All `class Config:` should be replaced with newer format `model_config = ConfigDict(...)`